electrum/lnsweep.py

```diff
@@ -297,11 +297,13 @@ def create_txns_for_htlc(
         ctn=ctn)
     for (direction, htlc), (ctx_output_idx, htlc_relative_idx) in htlc_to_ctx_output_idx_map.items():
         if direction == RECEIVED:
-            if chan.lnworker.get_payment_status(htlc.payment_hash) == PR_PAID:
-                preimage = chan.lnworker.get_preimage(htlc.payment_hash)
-            else:
+            if chan.lnworker.is_incomplete_mpp(htlc.payment_hash):
```

Comment:
another note is that this line duplicated 130 lines below, because of the symmetry, and both instances need to be changed, not just this one